contrib/fuzz/unstable_call_graph.py

```diff
@@ -0,0 +1,235 @@
+        type=bool,
+        default=False)
+
+    args = parser.parse_args()
+    
+    unstable_edges = []
+    unstable_functions = set()
+    strings_to_skip = set()
+
+    with open(args.afl_fuzzer_stats, 'r') as fuzzer_stats_file:
+        for line in fuzzer_stats_file:
+            if line.startswith('var_bytes'):
+                unstable_edges = line.split(":")[1].strip().split(" ")
+                break
+
+    with open(args.afl_edge_id_file, 'r') as edge_id_file:
+        for line in edge_id_file:
+            for edge in unstable_edges:
+                if f'edgeID={edge}' in line:
+                    function = line.strip().split(" ")[1].split("=")[1]
+                    if function not in unstable_functions:
+                        unstable_functions.add(function)
+
+    if args.dump_unstable:
+        with open(args.dump_unstable, 'w') as unstable_file:
+            for function in unstable_functions:
+                unstable_file.write(function + '\n')
+    
+    if args.skip_strings_file:
+        with open(args.skip_strings_file, 'r') as skip_file:
+            strings_to_skip = set(line.strip() for line in skip_file if line.strip())
+
+    try:
+        lines = check_output(["objdump", "-d", args.binary_file]).splitlines()
+    except:
+        exit()
+
+    G = nx.DiGraph()
+    curFunc = None
+    architecture = get_architecture(args.binary_file)
+
+    if architecture == 'x86_64':
+        call_pattern = r'^.*\bcall\s+([0-9a-zA-Z])+\s+<(.*)>$'
+    elif architecture == 'arm64':
+        call_pattern = r'^.*\bbl\s+([0-9a-zA-Z])+\s+<(.*)>$'
+    else:
+        raise ValueError("Unsupported architecture: " + architecture)
+
+    for l in lines:
+        l = l.decode('utf-8')
+        m = re.match(r'^([0-9a-zA-Z]+)\s+<(.*)>:$', l)
+        if m:
+            curFunc = m.group(2)
+            continue
+
+        if curFunc == None:
+            continue
+
+        m = re.match(call_pattern, l)
+        if m:
+            G.add_edge(curFunc, m.group(2))
+
+    print(f"Graph has {G.number_of_nodes()} nodes and {G.number_of_edges()} edges.")
+    
+    seen_edges = set()
+
+    def create_hierarchy(G, root):
+        demangled = demangle(root)
+        node = {
+            demangled: {
+                'children': {},
+                'is_unstable': root in unstable_functions,
+            }
+        }
+        for child in G.successors(root):
+            edge = (root, child)
+            if edge in seen_edges:
+                continue
+            seen_edges.add(edge)
+            
+            demangled_child = demangle(child)
+            if child in unstable_functions and any(demangled_child.startswith(string) for string in strings_to_skip):
+                continue            
+            
+            node[demangled]['children'] = node[demangled]['children' | create_hierarchy(G, child)
+        return node
+
+    target_node = next((node for node in G.nodes() if args.target_function in node), None)
```

Comment:
```suggestion
    target_node = next((node for node in G.nodes() if args.target_function == node), None)
```

I think an equality check might be better. I tried this script on a really simple example that had short function names e.g. "A", "B", "C" and when I tried setting the target function to "A" it would pick the first symbol that contained an "A" instead of the function "A".